utf8proc.h

```diff
@@ -45,8 +43,11 @@
  *    (i.e. UTF-8 encoded UTF-16 surrogates)
  *  - support for korean hangul characters
  *  Unicode Version 7.0.0 is supported.
```

Comment:
we should add either a newline or another `-` here, otherwise this is showing up on the same line as "support for korean hangul characters"